example-workflow/workflow.yaml

```diff
@@ -0,0 +1,94 @@
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        pmlogger_interval:
+          display:
+            description: The logger collection interval for PCP pmlogger
+            name: PCP pmlogger collection interval
+          type:
+            type_id: integer
+        sysbench_threads:
+          display:
+            description: The number of threads sysbench will run
+            name: sysbench threads
+          type:
+            type_id: integer
+        sysbench_events:
+          display:
+            description: The number of events sysbench will run
+            name: sysbench events
+          type:
+            type_id: integer
+        sysbench_cpumaxprime:
+          display:
+            description: The upper limit of the number of prime numbers generated
+            name: sysbench cpu max primes
+          type:
+            type_id: integer
+        sysbench_runtime:
+          display:
+            description: The total runtime in seconds for the sysbench tests
+            name: sysbench runtime seconds
+          type:
+            type_id: integer
+        elastic_host:
+          display:
+            description: The host URL for the ElasticSearch service
+            name: elasticsearch host url
+          type:
+            type_id: string
+        elastic_username:
+          display:
+            description: The username for the ElasticSearch service
+            name: elasticsearch username
+          type:
+            type_id: string
+        elastic_password:
+          display:
+            description: The password for the ElasticSearch service
+            name: elasticsearch password
+          type:
+            type_id: string
+        elastic_index:
+          display:
+            description: The index for the ElasticSearch service
+            name: elasticsearch index
+          type:
+            type_id: string
+steps:
+  pcp:
+    plugin: quay.io/dustinblack/arcaflow-plugin-pcp-test:latest
+    step: start-pcp
+    input:
+      pmlogger_interval: !expr $.input.pmlogger_interval
+      run_duration: !expr $.input.sysbench_runtime
+  sysbench:
+    plugin: quay.io/arcalot/arcaflow-plugin-sysbench:latest
```

Comment:
Pinning the plugins to specific tags, rather than latest, may be helpful since I think it's more important for the latest version to work than for it to be the most up to date.

The only exception is when the engine no longer supports a feature used by a plugin, but eventually we'll get to the point where the engine has backwards compatibility.

Reply:
I'd like to modify the tags to point to release versions, but we'll need to get a working set of plugin releases available first.